dns/bind/src/opnsense/service/templates/OPNsense/Bind/named.conf

```diff
@@ -138,14 +138,14 @@ zone "rpzbing" { type master; file "/usr/local/etc/namedb/primary/bing.db"; noti
 {%     set allow_query = helpers.getUUID(domain.allowquery) %}
 zone "{{ domain.domainname }}" {
         type {{ domain.type }};
-{%       if domain.type == 'slave' %}
+{%       if domain.type == 'secondary' %}
 {%         if domain.transferkey is defined %}
-        masters { {{ domain.masterip.replace(',', ' key "' ~ domain.transferkeyname ~ '"; ') }} key "{{ domain.transferkeyname }}"; };
+        masters { {{ domain.primaryip.replace(',', ' key "' ~ domain.transferkeyname ~ '"; ') }} key "{{ domain.transferkeyname }}"; };
 {%         else %}
-        masters { {{ domain.masterip.replace(',', '; ') }}; };
+        masters { {{ domain.primaryip.replace(',', '; ') }}; };
```

Comment:
There is no updated `masters` config value yet, this is coming in a future release. I did update the rest of the references so it's more consistent in the UI (and this can be a simple update once it's ready).